source/hic_hal/nxp/lpc11u35/uart.c

```diff
@@ -99,6 +99,9 @@ int32_t uart_set_configuration(UART_Configuration *config)
     baudrate = config->Baudrate;
     // Compute baud rate dividers
     mv = 15;
+#ifdef BOARD_RTL8195AM
+    config->Baudrate = 38400;
```

Comment:
I don't know if it's a bug or it's simply designed that way. The DAP UART on RTL8195AM simply does not work with baudrate other than 38400. There is no specific doc on this.